### PR TITLE
feat(core): add `EntityRepository.getEntityName()` method

### PR DESCRIPTION
**Pull request by ruscon**

### CODE REVIEW

File: packages/core/src/entity/EntityRepository.ts [hunk 1]
```diff
@@ -286,6 +286,10 @@ export class EntityRepository<T extends object> {
     return this._em;
   }
 
+  getEntityName(): EntityName<T> {
+    return this.entityName;
```
Comment (B4nan) [inline packages/core/src/entity/EntityRepository.ts:290]: maybe we should always return string here? the type could stay the same for better inference support

Comment (ruscon) [reply packages/core/src/entity/EntityRepository.ts:290]: It would be better to return `Constructor<T>`, but I understand this is not possible now :)

Comment (ruscon) [reply packages/core/src/entity/EntityRepository.ts:290]: In any case, I don’t know what cases other people will have.

Those who need a string can use `Utils.className(this.getRepository().getEntityName())`

Comment (B4nan) [reply packages/core/src/entity/EntityRepository.ts:290]: It's supposed to be _entity name_, so I think a string is actually what most people would expect. While we accept both string name and class reference, the getters should be consistent in what they return.

If you need the class reference, you can always get that from the metadata, e.g. `em.getMetadata('Foo').class`.

Comment (ruscon) [reply packages/core/src/entity/EntityRepository.ts]: Ok, done

File: packages/core/src/entity/EntityRepository.ts [hunk 2]
```diff
@@ -286,8 +286,8 @@ export class EntityRepository<T extends object> {
     return this._em;
   }
 
-  getEntityName(): EntityName<T> {
-    return this.entityName;
+  getEntityName(): string {
+    return Utils.className(this.entityName)
```
Comment (B4nan) [inline packages/core/src/entity/EntityRepository.ts]:
```suggestion
    return Utils.className(this.entityName);
```

**Comment by ruscon** [comment]
Please make a new npm version when you have time. I want to test it. Thanks.

**Comment by B4nan** [comment]
every commit to master is published to npm as dev version

**Pull request merged**